mopidy/local/__init__.py

```diff
@@ -24,7 +24,7 @@ def get_config_schema(self):
         schema['library'] = config.String()
         schema['media_dir'] = config.Path()
         schema['data_dir'] = config.Path()
-        schema['playlists_dir'] = config.Path()
+        schema['playlists_dir'] = config.Deprecated()
```

Comment:
Probably not worth it in this case, but we could probably also do something like to a `config.Deprecated(msg='...')` or `config.Deprecated(new='m3u/data_dir')` to ease migration.